BookPlayer/Library/ItemListViewController.swift

```diff
@@ -138,9 +143,24 @@ class ItemListViewController: UIViewController, ItemList, ItemListAlerts, ItemLi
         self.animateView(self.bulkControls, show: editing)
         self.tableView.setEditing(editing, animated: true)
 
+        if editing {
+            previousLeftButtons = navigationItem.leftBarButtonItems
+            navigationItem.leftBarButtonItems = [selectButton]
+            selectButton.isEnabled = tableView.numberOfRows(inSection: 0) > 0
```

Comment:
let's use the enum `Section` that we have, replacing `inSection: 0)` for `inSection: Section.data.rawValue)`